mdoc/Mono.Documentation/Updater/Statistics/StatisticsCollector.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Mono.Documentation.Updater.Frameworks;
+
+namespace Mono.Documentation.Updater.Statistics
+{
+    public class StatisticsCollector
+    {
+        public Dictionary<string, StatisticsStorage> Storages { get; } =
+            new Dictionary<string, StatisticsStorage>();
+
+        public void Increment(string framework, StatisticsItem item, StatisticsMetrics metrics, int delta = 1)
```

Comment:
I think a better member name than `Increment`, would be something like `AddMetric`. Mainly because calling `Increment` and passing in a `Removed` just _sounds_ kind of weird.